src/pkg/packager/common_test.go

```diff
@@ -132,7 +133,7 @@ func TestValidatePackageArchitecture(t *testing.T) {
 				return true, nodeList, nil
 			})
 
-			err := p.validatePackageArchitecture()
+			err := p.validatePackageArchitecture(context.TODO())
```

Comment:
Should this be switched to a normal context?

Reply:
Changed to use `context.Background()`